apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-content-palette/dot-content-palette.component.scss

```diff
@@ -7,6 +7,19 @@
     flex-direction: column;
     padding: $spacing-1;
     border-left: 1px solid $gray-lighter;
+    background: $white;
+    position: relative;
+    transition: all $basic-speed ease-in-out;
+
+    &.collapsed{
+        width: 0px;
+        min-width: 0px;
+        padding: 0px;
```

Comment:
This animation should be a slide in / out, not a scale to hide.

You might nee to use `translate`.